docs/src/main/sphinx/functions/string.rst

```diff
@@ -258,6 +258,17 @@ String functions
 
     Removes leading and trailing whitespace from ``string``.
 
+.. function:: trim( [ [ {LEADING | TRAILING | BOTH} ] [ string ] FROM ]
+                    charValueExpression ) -> varchar
+
+    Removes specified whitespace or string from ``charValueExpression`` at
+    ``string``, where ``charValueExpression`` can be a quoted string
+    or a function::
```

Comment:
I think there's no need to mention that the argument accepts functions. Existing functions don't have such explanation.